engine/src/main/java/com/datatorrent/stram/util/LoggerUtil.java

```diff
@@ -355,4 +364,88 @@ private static boolean isErrorLevelEnable()
     return false;
   }
 
+  /**
+   * Adds Logger Appender
+   * @param name Appender name
+   * @param properties Appender properties
+   */
+  public static void addAppender(String name, Properties properties)
+  {
+    if (getAppendersNames().contains(name)) {
+      logger.warn("A logger appender with the name '{}' exists. Cannot add a new logger appender with the same name", name);
+    } else {
+      try {
+        Method method = PropertyConfigurator.class.getDeclaredMethod("parseAppender", Properties.class, String.class);
+        method.setAccessible(true);
+        Appender appender = (Appender)method.invoke(new PropertyConfigurator(), properties, name);
```

Comment:
Agree, I can. But I need to keep the method object as a static variable and I want to avoid it. I don't think the fetching of the method can be a very expensive call, even if we need to do it for setting of several appenders.